tests/lightning_qubit/test_eval_jaxpr.py

```diff
@@ -0,0 +1,413 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module tests the default qubit interpreter.
+"""
+from this import d
+import pytest
+
+import pennylane as qml
+
+jax = pytest.importorskip("jax")
+
+from jax import numpy as jnp  # pylint: disable=wrong-import-position
+
+
+
+# must be below the importorskip
+# pylint: disable=wrong-import-position
+
+
+@pytest.fixture(autouse=True)
+def enable_disable_plxpr():
+    qml.capture.enable()
+    yield
+    qml.capture.disable()
```

Comment:
For future proofing, can you move this fixture to `conftest.py` and set `scope="function"`? Then you can use the following instead:
```python
pytestmark = pytest.mark.usefixtures("enable_disable_plxpr")
```

Reply:
I rather do that when we start to have more than one file with plxpr tests.

Reply:
Sounds good, resolving.